tests/results/longevity/1.2.0/1.2.0.md

```diff
@@ -0,0 +1,238 @@
+# Results
+
+Note: product telemetry feature was enabled but sending of product telemetry data was disabled.
+
+## NGINX OSS
+
+### Test environment
+
+NGINX Plus: false
+
+GKE Cluster:
+
+- Node count: 3
+- k8s version: v1.27.8-gke.1067004
+- vCPUs per node: 2
+- RAM per node: 4022900Ki
+- Max pods per node: 110
+- Zone: us-central1-c
+- Instance Type: e2-medium
+
+NGF pod name -- ngf-longevity-nginx-gateway-fabric-7f596f74c5-xzkzb
```

Comment:
I see value in them. Because we can look for logs and metrics if we know the name of the pod. For example, some time after we finished running a test  (to follow up on some investigation). Or a reviewer can look through logs or see metrics for themselves if they have some questions.

Reply:
We should add it to the automation.

Reply:
> We should add it to the automation.

kk.

there is also a bug on grabbing errors from NGF logs.
And I think log queries will not differentiate between NGINX and NGINX Plus

I will add an issue with improvements and fixes